superset/cli.py

```diff
@@ -40,6 +41,14 @@
 logger = logging.getLogger(__name__)
 
 
+feature_flags = config.DEFAULT_FEATURE_FLAGS.copy()
+feature_flags.update(config.FEATURE_FLAGS)
+feature_flags_func = config.GET_FEATURE_FLAGS_FUNC
+if feature_flags_func:
+    # pylint: disable=not-callable
+    feature_flags = feature_flags_func(feature_flags)
```

Comment:
This breaks the CLI when `feature_flags_func` uses `g.user` or any other app context based variables.

```
RuntimeError: Working outside of application context.
```

Reply:
I think we can just always add the command but do nothing or throw a warning is a feature is not turned on, then we can use the default `feature_flag_manager` as normal.

The part where you initialize feature flags differently is probably not needed.

Reply:
I added a quick workaround in #12088.